src/vuestic-theme/vuestic-components/vuestic-progress-bar/VuesticProgressBar.vue

```diff
@@ -23,72 +31,39 @@
       CircleBar
     },
     props: {
-      type: {
-        type: String,
-        default: 'horizontal'
-      },
-      min: {
-        type: Number,
-        default: 0
-      },
-      max: {
+      value: {
         type: Number,
         default: 100
       },
-      size: {
+      color: {
         type: String,
-        default: 'basic'
+        default: 'primary'
```

Comment:
Primary - is not a color but more like type.

I suggest 
* `type` -> `isVertical` (default: false)
* `theme` / `type` -> (primary, danger etc)
* `color` and `backgroundColor` either make optional or remove altogether, user can override these via css.